content/en/monitors/guide/division-in-monitor-evaluations.md

```diff
@@ -0,0 +1,211 @@
+
+```
+| Timestamp             | Value    |
+| :-------------------- | :------- |
+| 2019-03-29 11:00:00   | 1        |
+| 2019-03-29 11:00:30   |          |
+| 2019-03-29 11:01:00   | 2        |
+| 2019-03-29 11:01:30   | 1        |
+| 2019-03-29 11:02:00   |          |
+```
+
+**Modified**: `sum:my_metric.is.sparse{*}.rollup(sum,60)`
+
+```
+| Timestamp             | Value    |
+| :-------------------- | :------- |
+| 2019-03-29 11:00:00   | 1        |
+| 2019-03-29 11:01:00   | 3        |
+| 2019-03-29 11:02:00   | 1*       |
+```
+
+The `rollup()` function will create time buckets based on time intervals you define, which can be useful for ignoring "gaps" in your data if you set a rollup interval greater than the length of the gaps in your metric. In this case, the new buckets are the sums of the values in a 60 second window.
+
+\*Notice that the value at timestamp `2019-03-29 11:02:00` for the modified query doesn't align with the value in the chart above. This is because the `.rollup()` function is aligned to UNIX time. In this case, we can assume there is a value of `1` at `2019-03-29 11:02:30` which is in the rollup window, but not the monitor evaluation window. To avoid triggering your monitor based on an incomplete rollup interval containing only a small sample of data, you should add an **Evaluation Delay** to the monitor of at least the length of the rollup interval.
+
+## `.fill()`
+
+You can apply a `.fill()` function to ensure all time buckets have valid values. For **gauge** metric types, the default interpolation is linear or `.fill(linear)`. For **count** and **rate** type metrics, the default is `.fill(null)`, which disables interpolation.
+
+**Original**: `sum:my_metric.misaligned.groups.rate{env:a} by {timer,env}`
+
+```
+| Timestamp             | timer:1min,env:a    | timer:2min,env:a    |
+| :-------------------- | :------------------ | :------------------ |
+| 2019-03-29 12:00:00   | 1                   | 1                   |
+| 2019-03-29 12:01:00   | 1                   |                     |
+| 2019-03-29 12:02:00   | 1                   | 1                   |
+| 2019-03-29 12:03:00   | 1                   |                     |
+| 2019-03-29 12:04:00   | 1                   | 1                   |
+```
+
+Let's assume that `my_metric.misaligned.groups` is metric type **rate** so there is no interpolation as default. Consider this query:
+
+```
+sum(last_5m):sum:my_metric.misaligned.groups.rate{timer:1min,env:a} / sum:my_metric.misaligned.groups.rate{timer:2min,env:a}
+```
+
+You would see the following
+
+| Path                | Evaluation                          | Result |
+| :------------------ | :---------------------------------- | :----- |
+| `classic_eval_path` | **1/1 + 1/Nan + 1/1 + 1/Nan + 1/1** | 3      |
+
+By adding interpolation, you can ensure that metrics at every time interval
+
+**Modified**: `sum:my_metric.misaligned.groups.rate{env:a} by {timer,env}.fill(last)`
```

Comment:
When `RATE` metrics are queried with either `.as_count()` or `.as_rate()`, they are essentially treated like counts during time aggregation (even though they are stored as a per second rate). If the query has `.as_rate()`, they are converted back to a per second rate after time aggregation. If you query a `RATE` metric without either of these modifiers, it's essentially treated like a gauge, and that works since it's stored as a per second rate, but we don't officially support doing that, and it may not work in the future.

I would recommend always using one of these modifiers for `COUNT`/`RATE` metrics (like the UI query editor does), and not using interpolation.